modules/sdk-coin-sui/package.json

```diff
@@ -46,7 +46,9 @@
     "bignumber.js": "^9.0.0",
     "bs58": "^4.0.1",
     "js-sha3": "^0.8.0",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
```

Comment:
No package lock changes corresponding to the added dependencies?

Reply:
the same version is listed in `yarn.lock`